### ISSUE TITLE
searchgui tide glibc error

### ISSUE THREAD

**Issue by bernt-matthias** [compomics/searchgui#182]
One of the search engines (tide) seems to need glibc:

```
/gpfs1/work/songalax/galaxy-dev/database/jobs_directory/005/5277/working/bin/resources/Tide/linux/linux_64bit/crux: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by /gpfs1/work/songalax/galaxy-dev/database/jobs_directory/005/5277/working/bin/resources/Tide/linux/linux_64bit/crux)
```

Unfortunatelly we execute on a cluster system an can not update / change the glibc.

Can this be fixed? Maybe link statically?

see also: https://github.com/bioconda/bioconda-recipes/issues/11000

**Comment by hbarsnes** [comment]
I'd recommend contacting the Tide developers directly about this. You can do this via the Crux users google group: https://groups.google.com/forum/#!forum/crux-users.

**Comment by bernt-matthias** [comment]
Dear @hbarsnes I have some ideas on potential solutions: https://github.com/bioconda/bioconda-recipes/issues/11000 .. would be nice to get your feedback.

**Comment by hbarsnes** [comment]
@bernt-matthias We'd be happy to update the Tide/Crux binaries in SearchGUI as long as this does not reduce the amount of use cases in which Tide can be executed directly from SearchGUI.

Let me know if you manage to come up with a generic solution that seems to work.

**Comment by CarlosHorro** [comment]
No, I didn't have that problem when I was testing on Linux & Mac, although I suppose it depends on every software configuration...

**Comment by hbarsnes** [comment]
@bernt-matthias Can we close this one?

**Comment by bernt-matthias** [comment]
Ideally we would have the requirements like tide in the conda recipe requirements (currently its only xtandem).

**Comment by hbarsnes** [comment]
Ok, we'll leave it open then.

**Comment by bernt-matthias** [comment]
To let you know: the following is available in conda-forge or bioconda:

- Comet	https://anaconda.org/bioconda/comet-ms
- makeblastdb	https://anaconda.org/bioconda/blast-legacy
- MetaMorpheus	https://anaconda.org/conda-forge/metamorpheus
- MSGF	https://anaconda.org/bioconda/msgf_plus
- OMSSA	https://anaconda.org/bioconda/omssa
- ThermoRawFileParser	https://anaconda.org/bioconda/thermorawfileparser
- XTandem	https://anaconda.org/bioconda/xtandem

Unavailable so far are: 

- Andromeda X
- directag
- 'MS Amanda'
- MyriMatch
- Novor
- Tide

The available might be easily provided by conda given that the required versions are available and that SearchGUI does not use hard coded paths to the binaries. Might be a nice project to replace them step by step. Would also reduce the (conda) package size dramatically.
